telosnetwork/telos-wallet
# repairing the total balance calculation

# Fixes #94 

## Description
The calculation was fixed as described in issue #94. The display part was not implemented due to a previous discussion where we agreed to skip that part because it will be refactored anyway in the near future.

## Test scenarios
Let's call `myaccount` any account you control with the following balances:
- has some TLOS staking for NET and CPU
- some refunding TLOS (when you unstake)
- some TLOS staked in REX with unclaimed profits
- some liquid deposits (funds) in REX,
- some liquid in EVM,
- and finally some liquid TLOS. 

### test scenarios 1
- Log in using  `myaccount` in web-wallet
- explore the same account using OBE.
- The total balance should only differ in Liquid EVM amount

### test scenarios 2
- Log in using  `myaccount` in web-wallet
- Move any TLOS from any of the balances without losing ownership (do not transfer)
- Refresh the website
- The total balance should be the same as before the balance movement.

## Checklist:
